config/foreman-proxy-content.migrations/200616155948-disable-pulp-3-proxying.rb

```diff
@@ -0,0 +1,15 @@
+FOREMAN_PROXY_CONTENT = 'foreman_proxy_content'.freeze
+PROXY_YUM = 'proxy_pulp_yum_to_pulpcore'.freeze
+PROXY_ISOS = 'proxy_pulp_isos_to_pulpcore'.freeze
+
+def upgrade
+  answers[FOREMAN_PROXY_CONTENT] = {} unless answers[FOREMAN_PROXY_CONTENT].is_a?(Hash)
+  answers[FOREMAN_PROXY_CONTENT][PROXY_YUM] = false
+  answers[FOREMAN_PROXY_CONTENT][PROXY_ISOS] = false
+end
+
+if answers[FOREMAN_PROXY_CONTENT] &&
+   answers[FOREMAN_PROXY_CONTENT][PROXY_YUM].nil? &&
```

Comment:
I used this pattern previously when I was confident that answers[FPC] should already be a hash, but that wasn't the case for answers in this scenario previously, opening the possibility of a NoMethodError:

```
irb(main):012:0> answers
=> {"FPC"=>true}
irb(main):013:0> if answers['FPC']
irb(main):014:1>   puts "hello"
irb(main):015:1> end
hello
=> nil
irb(main):016:0> if answers['FPC'] && answers['FPC']['PROXY'].nil?
irb(main):017:1>   puts "hello"
irb(main):018:1> end
Traceback (most recent call last):
        2: from lib/tasks/console.rake:5:in `block in <top (required)>'
        1: from (irb):16
NoMethodError (undefined method `[]' for true:TrueClass)
irb(main):019:0> 
```

Reply:
Hmmm, my usage may have been incorrect as well. I'm investigating it a bit further.